root/static/scripts/edit/URLCleanup.js

```diff
@@ -1929,7 +1929,26 @@ const CLEANUPS = {
     match: [new RegExp('^(https?://)?(www\\.)?mainlynorfolk\\.info', 'i')],
     type: LINK_TYPES.otherdatabases,
     clean: function (url) {
-      return url.replace(/^(?:https?:\/\/)?(?:www\.)?mainlynorfolk\.info\//, 'https://mainlynorfolk.info/');
+      url = url.replace(/^(?:https?:\/\/)?(?:www\.)?mainlynorfolk\.info\//, 'https://mainlynorfolk.info/');
+      // Drop unneeded index.html from artist pages
+      url = url.replace(/^https:\/\/mainlynorfolk\.info\/([^/]+)\/index\.html$/, 'https://mainlynorfolk.info/$1/');
+      return url;
+    },
```

Comment:
This part of the second commit would better fit the first commit which is about normalization (https & -www).
By the way, the first commit message should probably not misleadingly specify “to HTTPS” in header.
I just removed "and normalize to HTTPS" from ticket’s summary as it also drops www and index.html and as normalization is just the usual prerequisite to validation.


The comment in code here is probably not needed.